CredentialProvider/CredentialProviderFennec.entitlements

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
+	<true/>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.org.mozilla.ios.Fennec</string>
+	</array>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)org.mozilla.ios.Fennec</string>
+	</array>
+</dict>
+</plist>
```

Comment:
Do we need two additional copies of this file for _Firefox_ and _Firefox Beta_? I see the other App Extensions use a shared `Extensions/Entitlements/*` folder - can we just add the `.autofill-credential-provider` entitlement to those? (Not sure if that is valid and if that passes app review checks)

Reply:
We do need additional copies and I just added a commit to this branch to do that.